pmc/challenges/default.json

```diff
@@ -144,10 +144,10 @@
         {
           "operation": {
             "operation-type": "create-index",
-            "settings": {
+            "settings": {%- if index_settings is defined %} {{index_settings | tojson}} {%- else %} {
               "index.sort.field": "timestamp",
               "index.sort.order": "desc"
-            }
+            }{%- endif %}
           }
```

Comment:
Do we need to add something similar to "settings" at line 195?
```
 "operation": {
            "operation-type": "create-index",
            "settings": {
              "index.refresh_interval": "30s",
              "index.number_of_shards": 6,
              "index.translog.flush_threshold_size": "4g"
            }
```

Reply:
Oh, good catch! I've pushed a fix in 97783f8.